Server/Components/Devices/ChatCard.razor.cs

```diff
@@ -22,18 +23,20 @@ public partial class ChatCard : AuthComponentBase, IDisposable
     public required ChatSession Session { get; set; }
 
     [Inject]
```

Comment:
Reading up on `inject`, interesting, we can't instantiate razor.cs classes using constructor-based dependency injection?

Reply:
Looks like I forgot to respond to this.  Yeah, components must have parameterless constructors.